### DISCUSSION TITLE
universal display driver in user_config_override.h how to do that?

### DISCUSSION THREAD

**Discussion by mascheck55** [arendst/Tasmota#22356] [Q&A] [answered]
I am new and stumbled into the trap of universal display driver, one of the options is 4. compile it into the binary. I have no idea how to build this section there in the user_config_override.h. Is there somewhere an example.
Thanks for any hint!

**Comment by sfromis** [comment, marked as answer]
I think that it is like the example found here:
https://github.com/arendst/Tasmota/blob/development/tasmota/displaydesc/readme.md

**Reply by mascheck55** [reply]
Thank you, that is what I was seeking. Should there also named the GPIO Pins for I2C as replacement for the three stars? Similar to the Displayaddress 3c?      :H,SH1106,128,64,1,I2C,3c,*,*,*\n"

**Reply by sfromis** [reply]
No, there *should* not. The point is that when the display descriptor has `*,*,*` this will refer to you having the relevant gpios configured in the template, which is what I'd suggest to do.

Still, you *could* also hardcode the gpios in the display descriptor, but I see it as good practice to avoid this.
